lib/core/src/lib/notifications/components/notification-history.component.html

```diff
@@ -7,7 +7,7 @@
             id="adf-notification-history-open-button"
             (menuOpened)="onMenuOpened()">
         <mat-icon matBadge="&#8288;"
-                  [matBadgeHidden]="!notifications.length"
+                  [matBadgeHidden]="!unreadNotifications.length || badgeHidden"
```

Comment:
the `badgeHidden` is not needed, it's already covered by `unreadNotifications.length`

Reply:
removed badgeHidden